### ISSUE TITLE
[electron][macOS] The Preferences menu should belong to the main application menu not File

### ISSUE THREAD

**Issue by kittaakos** [eclipse-theia/theia#6497]
### Description
 

As a macOS user, I would expect the ~`Preferences`~ `Settings` menu item under the `0` index, not under `File`. See, it is the same in VS Code:

![Screen Shot 2019-11-05 at 17 30 23](https://user-images.githubusercontent.com/1405703/68226352-3782d380-fff2-11e9-9a1c-f3a998550921.jpg)

In a Theia-based app, the `Preferences` in under `File`:

![Screen Shot 2019-11-05 at 17 32 03](https://user-images.githubusercontent.com/1405703/68226393-48334980-fff2-11e9-8013-7c4f3ade9d8f.jpg)

However, on Windows and Linux, it should go under `File`:
![Screen Shot 2019-11-05 at 17 33 52](https://user-images.githubusercontent.com/1405703/68226532-86c90400-fff2-11e9-880d-c2ce14738253.jpg)

Note, `Open Preferences` should be renamed as well.

### Reproduction Steps

**OS and Theia version:**

**Diagnostics:**

**Comment by joewiz** [comment]
Agreed. As a longtime Mac user, I find it confusing that Theia-based apps lack a Preferences item inside the app menu. As an example of the correct behavior, see these cross-platform apps:

**Chrome**

> <img width="316" alt="Screen Shot 2020-09-08 at 3 36 20 PM" src="https://user-images.githubusercontent.com/59118/92520096-1332ed00-f1e9-11ea-8e37-eb5450956627.png">

**Discord**

> <img width="234" alt="Screen Shot 2020-09-08 at 3 38 50 PM" src="https://user-images.githubusercontent.com/59118/92520315-69079500-f1e9-11ea-8fca-b9a2a886fdaa.png">

**Atom**

> <img width="252" alt="Screen Shot 2020-09-08 at 3 39 26 PM" src="https://user-images.githubusercontent.com/59118/92520366-7c1a6500-f1e9-11ea-835b-b35c89e9c93f.png">

**Slack**

> <img width="234" alt="Screen Shot 2020-09-08 at 3 39 41 PM" src="https://user-images.githubusercontent.com/59118/92520397-850b3680-f1e9-11ea-9395-59680dba3edf.png">
